armi/reactor/components/basicShapes.py

```diff
@@ -17,8 +17,14 @@
 
 Many reactor components can be described in 2D by circles, hexagons, rectangles, etc. These
 are defined in this subpackage.
-"""
 
+.. impl:: ARMI shall support a reasonable set of basic shapes.
+   :id: IMPL_0_2_0
+   :links: REQ_0_2
```

Comment:
do we want to do numbering like this, or maybe use slightly more meaningful short names? I think names might be easier.

Reply:
We can pick some other name if you liked.

For "REQ_4" I thought "IMPL_4_X" and "TEST_4_X" would be really quick and easy to associate back to their requirements, for humans reading.

But you want the requirements to be short strings, rather than numbers?  We can do that.

Reply:
Okay, take a look. I have changed the requirements to be short strings, instead of numbers.

(Sorry, I'm just used to requirements being numbered. But I don't really have a preference.)